force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -147,7 +183,14 @@ public with sharing class VariablesDataTypesOperators {
      * @return True if the date is today or in the future, False otherwise.
      */
     public static Boolean isDateTodayOrFuture(Date dt) {
-        return null; // Replace null with the variable you used to store the result
+        Boolean result;
+        if (dt > Date.today()) {
+            result=true;
+        }
+            else {
+                result=false;
```

Comment:
for question #9 the else is under the bracket, in this one  it's a space forward, try to be consistent so people that are reading your code can know exactly what to expect.